examples/AzureSDKDemoSwift/Source/Common.swift

```diff
@@ -101,7 +101,7 @@ struct AppState {
                 logger: ClientLoggers.none
             )
             AppState.internalBlobClient = try? StorageBlobClient(
-                accountUrl: AppConstants.storageAccountUrl,
+                blobStorageEndpoint: AppConstants.storageAccountUrl,
```

Comment:
The name was changed to add clarity, as I was writing the documentation, and reading the blob storage service / portal / REST API documentation. I renamed it here so that it matched the same argument in the credentials, and the `endpoint(forAccount:)` method.

Calling it a (storage) account URL, I was assuming that it would be the form "<accountname>.core.windows.net" however that would not work here - what we actually need is the storage account's blob service endpoint "<accountname>.blob.core.windows.net". Also thinking ahead if we support other storage services, would we also have a parameter `accountURL` that gets passed to `StorageFileClient` too? That would be confusing since that URL would be different than the `accountURL` that gets passed to `StorageBlobClient`.

Reply:
Makes sense. Perhaps just `endpoint` or `withEndpoint` would be more concise?

Reply:
When I looked at the Java builder class it looks like they used `endpoint`. @anuchandy what are you using?

Reply:
I'm okay with `endpoint`

Reply:
Fixed in b19eccc